jctools-build/src/main/java/org/jctools/queues/atomic/JavaParsingAtomicArrayQueueGenerator.java

```diff
@@ -46,6 +47,8 @@
 import com.github.javaparser.ast.visitor.VoidVisitorAdapter;
 
 public final class JavaParsingAtomicArrayQueueGenerator extends VoidVisitorAdapter<Void> {
```

Comment:
Changes to this class were largely refactoring to try to stay similar to the linked queue generator